src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -328,32 +206,29 @@ and end events to the document:
 {% includecode content_path="web/fundamentals/design-and-ui/input/touch/_code/touch-demo-1.html" region_tag="handle-start-gesture" adjust_indentation="auto" %}
 </pre>
 
-The end callback we add is `handleGestureEnd` which removes the move
-and end events from the document when the gesture has finished:
+The end callback we add is `handleGestureEnd`, which removes the move
+and end event listeners from the document when the gesture has finished like so:
 
 <pre class="prettyprint">
 {% includecode content_path="web/fundamentals/design-and-ui/input/touch/_code/touch-demo-1.html" region_tag="handle-end-gesture" adjust_indentation="auto" %}
 </pre>
 
-Mouse events follow this same pattern since it’s easy for a user to
-accidentally move the mouse outside of the element, which results in the move
-events no longer firing. By adding the move event to the document, we'll continue to get mouse movements regardless of where they are on the page.
-
-You can use the
-[Show potential scroll bottlenecks](https://developer.chrome.com/devtools/docs/rendering-settings#show-potential scroll bottlenecks)
-feature in Chrome DevTools to show how the touch events behave:
+<div class="attempt-left">
+  <p>By following this pattern of adding the move event to the document, if the
+  user starts interacting with an element and moves their gesture outside of
+  the element, we'll continue to get mouse movements regardless of where they
+  are on the page, because the events are being received from the document.</p>
 
-<img src="images/scroll-bottleneck-devtool.png" alt="Enable Scroll Bottleneck in DevTools">
-
-![Illustrating Binding Touch Events to Document in touchstart](images/scroll-bottleneck.gif){: .attempt-right }
+  <p>This diagram demonstrates what the touch events are doing as we add the
+  *move* and *end* events to the document once a gesture begins.</p>
+</div>
 
-With this enabled you can see where touch events are bound and ensure your logic for adding
-and removing listeners is working as you'd expect.
+![Illustrating Binding Touch Events to Document in touchstart](images/scroll-bottleneck.gif)
 
 <div class="clearfix"></div>
 
 
-### Handle Multi-Element Interaction
+#### Handle Multi-Element Interaction
 
 If you expect your users to use multiple elements at once, you can add the
 move and end events listeners directly to the elements themselves. This
```

Comment:
The issue with `mouse` events also applies to pointer events for mouse devices (both are "uncaptured" by default).  You can either follow the same pattern of pointer adding handlers to the document, or you can just call `target.setPointerCapture(event.pointerId)` in your `pointerdown` handler to ensure that even if your mouse leaves the element, you'll continue to get `pointermove` events on the target you're listening to.